server/repository/src/test_db/mod.rs

```diff
@@ -0,0 +1,109 @@
+#[cfg(not(feature = "postgres"))]
```

Comment:
I had to add migration version number to setup test, while doing that needed to change helper methods behind feature flags, they were both in the same file and I found it quite hard to navigate (since they were also dimmed), so relocate sqlite, postgres specific code to their own respective files

Reply:
IMHO file was small enough gets harder to navigate if you have too many files...

Reply:
I found it really hard to make the change (and actually previous changes in this file) when feature flags were combined in one file, there is a bunch of warnings (for unused imports) etc.. that is hard to work with.

> IMHO file was small enough gets harder to navigate if you have too many files

I can see your point, I think it's the matter of preference based on the differences in habitual way IDE features are used. In this case in particular, splitting up the features into their own branches helped me a lot (no need to do inline import to avoid compilation warning etc..)